index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
```

Comment:
Please specify a 'viewport' meta element

Reply:
you caught most of the bugs and professionally reviewed your code, you did a great job 👍 .

you missed explaining why the viewport meta element needs to be specified, this will enable the developer to understand why you are requesting the changes and also learn from you

Reply:
Thanks for the feedback :bulb:. So a better comment would be something like this:
"Please specify a 'viewport' meta element. This gives the browser instructions on how to control the page's dimensions and scaling."
?

Reply:
Please specify a 'viewport' meta element. This gives the browser instructions on how to control the page's dimensions and scaling.